src/App.js

```diff
@@ -3,28 +3,71 @@ import './App.css';
 import movieData from './data.js';
 import { render } from 'react-dom';
 import Movies from './Movies.js';
+import Form from './Form.js';
+import Preview from './Preview.js'
+import tomato1 from './images/tomato1.svg';
+import tomato2 from './images/tomatos2.svg';
+import tomato3 from './images/tomato3.svg';
+import tomato4 from './images/tomato4.svg';
+import magnifyGlass from './images/grey-magnify-glass.svg'
+
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      movie: ''
     }
   }
 
   addMovies = () => {
-    this.setState({movies: [...movieData.movies]})
```

Comment:
Great for hard coding the data... need to fetch it from the API to change the state